yb-voyager/src/tgtdb/oracle.go

```diff
@@ -619,3 +634,24 @@ func (tdb *TargetOracleDB) getConnectionString(tconf *TargetConf) string {
 func (tdb *TargetOracleDB) MaxBatchSizeInBytes() int64 {
 	return 2 * 1024 * 1024 * 1024 // 2GB
 }
+
+func (tdb *TargetOracleDB) getColumnDataTypes(tableName string) (map[string]string, error) {
```

Comment:
do we still need this given that we're getting the source data types from debezium.